### DISCUSSION TITLE
My github account has been blocked or restricted for some reason.

### DISCUSSION THREAD

**Discussion by AsadbekRajabboyevv** [community/community#146490] [Other Feature Feedback, Questions, & Ideas]
My account with the username AsadbekRajabboyev has been blocked or restricted, but I don't know why, I had big projects in it, please help me restore my account, I would like the github team to help me
![image](https://github.com/user-attachments/assets/ec865282-c0fc-43b5-bc5c-e3b36b3ec8e7)

**Comment by davevad93** [comment]
Hi @AsadbekRajabboyevv ,

For account related issues it's recommended to contact support, raise a ticket [here](https://support.github.com/contact/account), select "`Other`" and explain your issue.
